crates/agentgateway/src/config.rs

```diff
@@ -345,9 +404,113 @@ pub fn parse_config(contents: String, filename: Option<PathBuf>) -> anyhow::Resu
 				)
 				.unwrap_or(Duration::from_secs(60 * 5)),
 		}),
+		#[cfg(feature = "pat")]
+		pat: {
+			let enabled = parse::<bool>("PAT_ENABLED")?.unwrap_or(false);
+			let max_days = parse::<i64>("PAT_MAX_EXPIRY_DAYS")?
+				.unwrap_or_else(|| default_pat_max_expiry_days())
+				.max(0);
+			let strict_jwt = parse::<bool>("PAT_STRICT_JWT")?.unwrap_or(true);
+			let require_for_ai_routes = parse::<bool>("PAT_REQUIRE_FOR_AI_ROUTES")?.unwrap_or(false);
+			let required_route_patterns = parse::<String>("PAT_REQUIRED_ROUTE_PATTERNS")?
+				.map(|s| s.split(',').map(|p| p.trim().to_string()).collect())
+				.unwrap_or_default();
+			crate::config::PatConfig {
+				enabled,
+				max_expiry_days: if max_days == 0 {
+					default_pat_max_expiry_days()
+				} else {
+					max_days
+				},
+				strict_jwt,
+				require_for_ai_routes,
+				required_route_patterns,
```

Comment:
I think generally this pattern can be achieved by attaching the policy to specific route rules. Though if you have 100s of routes you want to apply it to it may get tedious I supppose